contributing/documentation/building_the_manual.rst

```diff
@@ -1,17 +1,18 @@
 .. _doc_building_the_manual:
 
 Building the manual with Sphinx
-===============================
+###############################
```

Comment:
```suggestion
===============================
```

Not sure on the reason for this change, as other files use `=`. RST primer says it doesn't make a difference.